### PR TITLE
add WG announcement

### PR DESCRIPTION
**Pull request by ywang-gilead**
@danielinteractive please update the date for the WG announcement in `news.csv` if needed.

### CODE REVIEW

**Comment by danielinteractive** [comment]
thanks @ywang-gilead , I added a blog structure - please have a look, if looks good can merge

**Review by danielinteractive — APPROVED** [review]

File: blog/wg_announcement/index.qmd [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The primary objective of the SWE WG is to engineer R packages that implement important methods that are often utilized in biostatistics but still missing from our R toolbox. The first product is the recently-published R package implementing mixed models for repeated measures (MMRM): [mmrm](https://cran.r-project.org/package=mmrm). In particular, a critical advantage of this package over existing implementations is that it is at least 3-5 times faster and converges more reliably. It also provides a comprehensive set of features: users can specify a variety of covariance matrices, weight observations, fit models with restricted or standard maximum likelihood inference, perform hypothesis testing with Satterthwaite adjusted degrees of freedom, and extract the least square means estimates by using the [emmeans package](https://github.com/rvlenth/emmeans). We aim to establish the R package [mmrm](https://cran.r-project.org/package=mmrm) as a new standard within biostatistics and will communicate it broadly.
```
Comment (kkmann) [inline blog/wg_announcement/index.qmd:15]: replace ` We aim to establish the R package [mmrm](https://cran.r-project.org/package=mmrm) as a new standard within biostatistics` 

Why biostatistics? Wouldn't it be more inclusive to invite people to collaborate? I don't see how we could establish it as a standard ;)

Comment (kkmann) [inline blog/wg_announcement/index.qmd:15]: Make it clear that collaboration on mmrm is open to anyone - it is just initiated by the ASA WG?

**Review by kkmann — APPROVED** [review]
Love it, some minor comments :)

File: blog/wg_announcement/index.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielinteractive) [reply blog/wg_announcement/index.qmd:15]: ok removed the 3-5 times, but we have evidence that it is that much faster - just not on paper yet... so "faster" is ok.

Comment (danielinteractive) [reply blog/wg_announcement/index.qmd:15]: hm ok good point, reworded a bit to make it clear?

Comment (danielinteractive) [reply blog/wg_announcement/index.qmd:15]: added a sentence

**Pull request merged**